### ISSUE TITLE
Invoice creation from command line: All but the first invoice have wrong styling

### ISSUE THREAD

**Issue by apiening** [kevinpapst/kimai2#1973]
**Describe the bug**
When there are multiple invoices created via command line in one run, the first invoice is fine but all following invoices are missing most of the stile definitions. The text is there, but at the wrong position with wrong format.

**To Reproduce**
1. Create an invoice template based on **PDF document**.
2. Create some invoices based on this template from the command line, with **two projects** for example:
```bash
bin/console kimai:invoice:create --user=username --timezone=Europe/Berlin --project 1,2 --template="templatename" --start=from-date --end=to-date
```
**Screenshots**
**Footer section with correct style (first invoice):**
![Bildschirmfoto 2020-09-13 um 14 22 25](https://user-images.githubusercontent.com/2064875/93017914-8e6f1700-f5cc-11ea-88f1-82fc865b383f.png)
**Footer for every following invoice of the same run:**
![Bildschirmfoto 2020-09-13 um 14 23 27](https://user-images.githubusercontent.com/2064875/93017934-b2caf380-f5cc-11ea-84ea-20d203544773.png)

**Additional context**
 - Kimai version 1.10.1
 - PHP version 7.4.9

**Comment by kevinpapst** [comment]
It works fine with the CSV/JSON templates, but I guess I never tried the PDF export in the command line
A bit strange, but okay. Thanks for the report!

**Comment by apiening** [comment]
It looks to me like the styles defined with
```
{{ encore_entry_css_source('invoice-pdf')|raw }}
```
in the default PDF template are only processed for the first invoice. But I'm not perfectly sure if this really is the cause and why.
Maybe it is enough to close the PDF generator instance and create a new instance for every occurring invoice just to make sure that everything is reset to its defaults.

**Comment by kevinpapst** [comment]
If you have some Symfony experience, please try to debug it yourself. 
I am currently very busy and can't promise feedback on complex issues in time - especially if you want to use the invoice creation in your workflow.

Creating a new instance of the renderer would be a huge refactoring, I don't think this is the right way to move forward.
Lets find out what is wrong. 

You could temporarily intercept the pdf processing and work with the HTML by commenting this block: 
https://github.com/kevinpapst/kimai2/blob/master/src/Invoice/Renderer/PdfRenderer.php#L42-L47

This should give you a hint what is wrong with the styles.

**Comment by kevinpapst** [comment]
will be fixed in 1.12

you could fix it manually in your installation with the code in src/Twig/EncoreExtension.php (see https://github.com/kevinpapst/kimai2/pull/2097/files)
